loadtest/control/simplecontroller/controller.go

```diff
@@ -102,6 +102,11 @@ func (c *SimpleController) Run() {
 		}
 	}
 
+	if len(c.actions) == 0 {
+		<-c.stopChan
+		return
+	}
```

Comment:
Did you get the controller spinning? :stuck_out_tongue:

Reply:
Yes 😄